docs/src/main/asciidoc/_migration-guide-for-4.0.adoc

```diff
@@ -315,9 +315,9 @@ Changes for the child entries for this prefix, please refer the following tables
 [cols="<30,<~", options="header"]
 |===
 |Legacy properties | Morden properties
-|checkpoint-storage-account|processor.checkpoint-store.account-name
-|checkpoint-access-key|processor.checkpoint-store.account-key
-|checkpoint-container|processor.checkpoint-store.container-name
+|*checkpoint*-storage-account|*processor.checkpoint-store*.account-name
```

Comment:
It's better not separate `checkpoint-storage-account` , how about delete `*` here?